apps/fares/test/retail_locations_data_test.exs

```diff
@@ -20,6 +20,12 @@ defmodule Fares.RetailLocationsDataTest do
       end
     end
 
+    test "all locations have a non-nil name" do
+      for %Location{name: name} <- Data.get() do
+        assert is_binary(name)
+      end
+    end
```

Comment:
I know it's usually good advice to write a failing test before fixing a bug, and this test would have failed in this specific scenario... but I feel like we'll probably never see _this_ specific scenario again, and that's all the test covers. It's also testing the output of a live external service, which has its own issues of making the test suite a bit slower and more unreliable. I'd advocate for not adding this, but would like to hear what y'all think.

(in terms of _process_ that would have prevented this bug, did we check our code against the [changelog](https://www.mbta.com/developers/v3-api/changelog) before moving to the new API version? it does call out this `name` change)

Reply:
Our pacing with the GTFS Creator repo is not always in sync. We could indeed improve our protocol between teams here to avoid this issue in the future. @phildarnowsky and I have discussed having the API team run our dotcom tests, but beyond that a changelog review would be great.

Reply:
You don't have to update to new API versions: it's why we maintain the old ones rather than making arbitrary changes.

Reply:
@phildarnowsky where/when do API version updates happen on prod?